### ISSUE TITLE
Should #CounterTest be packaged in MyCounter-Test instead of MyCounter?  @ 0638b16 (PDF)

### ISSUE THREAD

**Issue by offray** [SquareBracketAssociates/LearningOOPWithPharo#48]
Following the subsection "5.4 Define a Test Case" the learner gets a warning by the code assistant, saying that the "Test class not in a package with name ending with '-Test'". Because this is a book for Pharo novices, such warning could raise some concern on learners. 

Should the example code reflect the suggestions of the code assistant to reduce warnings?

**Comment by Ducasse** [comment]
Yes definitively Can you open a ticket in Pharo bugtracker so that we can produce a list of rules for a novice profile.
